### ISSUE TITLE
Bot fails at startup: NotLoggedInException, ServerSideRequestThrottlingException, ServerBusyOrOfflineEception

### ISSUE THREAD

**Issue by andirapandi** [PokemonGoF/PokemonGo-Bot#3846]
I am new to this bot.

Setting up bot several times on different systems and using different accounts, I have not been able to get the bot to actually start. Accounts are working, I can collect items and catch Pokemon on phone.

Error is:

  File "pokecli.py", line 39, in <module>
    from pgoapi.exceptions import NotLoggedInException, ServerSideRequestThrottlingException, ServerBusyOrOfflineException

### Expected Behavior

Bot starts and logs in.

### Actual Behavior

Bot does not log in, seems not to work.

### Your config.json (remove your credentials and any other private info)

{
	"auth_service": "ptc",
	"username": "xxx",
	"password": "yyy",
	"location": "51.22xxxxx,6.77xxxxx",
	"gmapkey": "abc",
	"libencrypt_location": "",
	"tasks": [{
		"type": "HandleSoftBan"
	}, {
		"type": "SleepSchedule",
		"config": {
			"enabled": false,
			"time": "23:54",
			"duration": "9:46",
			"time_random_offset": "00:24",
			"duration_random_offset": "00:43"
		}
	}, {
		"type": "CollectLevelUpReward"
	}, {
		"type": "IncubateEggs",
		"config": {
			"longer_eggs_first": false
		}
	}, {
		"type": "UpdateLiveStats",
		"config": {
			"enabled": false,
			"min_interval": 10,
			"stats": ["uptime", "stardust_earned", "xp_earned", "xp_per_hour", "stops_visited"],
			"terminal_log": true,
			"terminal_title": true
		}
	}, {
		"type": "TransferPokemon"
	}, {
		"type": "EvolvePokemon",
		"config": {
			"evolve_all": "none",
			"first_evolve_by": "cp",
			"evolve_above_cp": 550,
			"evolve_above_iv": 0.82,
			"logic": "and",
			"evolve_speed": 20,
			"use_lucky_egg": false
		}
	}, {
		"type": "RecycleItems",
		"config": {
			"min_empty_space": 5,
			"item_filter": {
				"Pokeball": {
					"keep": 70
				},
				"Potion": {
					"keep": 1
				},
				"Super Potion": {
					"keep": 1
				},
				"Hyper Potion": {
					"keep": 15
				},
				"Revive": {
					"keep": 5
				},
				"Razz Berry": {
					"keep": 60
				}
			}
		}
	}, {
		"type": "CatchVisiblePokemon"
	}, {
		"type": "CatchLuredPokemon"
	}, {
		"type": "SpinFort"
	}, {
		"type": "MoveToFort",
		"config": {
			"lure_attraction": true,
			"lure_max_distance": 2000
		}
	}, {
		"type": "FollowSpiral",
		"config": {
			"diameter": 5,
			"step_size": 68
		}
	}],
	"map_object_cache_time": 5,
	"forts": {
		"avoid_circles": true,
		"max_circle_size": 50
	},
	"websocket_server": false,
	"walk": 4.02,
	"action_wait_min": 4,
	"action_wait_max": 9,
	"debug": false,
	"test": false,
	"health_record": true,
	"location_cache": false,
	"distance_unit": "km",
	"reconnecting_timeout": 25,
	"catch_randomize_reticle_factor": 1.0,
	"catch_randomize_spin_factor": 1.0,
	"logging_color": true,
	"catch": {
		"any": {
			"catch_above_cp": 0,
			"catch_above_iv": 0,
			"logic": "or"
		},
		"// Example of always catching Rattata:": {},
		"// Rattata": {
			"always_catch": true
		}
	},
	"release": {
		"any": {
			"release_below_cp": 300,
			"release_below_iv": 0.8,
			"logic": "or"
		},
		"// Example of always releasing Rattata:": {},
		"// Rattata": {
			"always_release": true
		},
		"// Example of keeping 3 stronger (based on CP) Pidgey:": {},
		"// Pidgey": {
			"keep_best_cp": 3
		},
		"// Example of keeping 2 stronger (based on IV) Zubat:": {},
		"// Zubat": {
			"keep_best_iv": 2
		},
		"// Also, it is working with any": {},
		"// any": {
			"keep_best_iv": 3
		},
		"// Example of keeping the 2 strongest (based on CP) and 3 best (based on IV) Zubat:": {},
		"// Zubat": {
			"keep_best_cp": 2,
			"keep_best_iv": 3
		}
	},
	"vips": {
		"Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate!": {},
		"any": {
			"catch_above_cp": 1200,
			"catch_above_iv": 0.9,
			"logic": "or"
		},
		"Lapras": {},
		"Moltres": {},
		"Zapdos": {},
		"Articuno": {},

		"// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
		"Mewtwo": {},
		"Dragonite": {},
		"Snorlax": {},
		"// Mew evolves to Mewtwo": {},
		"Mew": {},
		"Arcanine": {},
		"Vaporeon": {},
		"Gyarados": {},
		"Exeggutor": {},
		"Muk": {},
		"Weezing": {},
		"Flareon": {}
	},
	"catch_throw_parameters": {
		"excellent_rate": 0.08,
		"great_rate": 0.42,
		"nice_rate": 0.2,
		"normal_rate": 0.3,
		"spin_success_rate": 0.7
	}
}

### Steps to Reproduce

This is the first time I have been using the bot, so it is a fresh install, according to installation page.

I tried both on Windows 8.1 and Ubuntu 14.04, same result each time.

I tried with 2 accounts, one that I use regularly without botting, and when getting the error, I confirmed that it still works on my Android phone, so account seems fine.

I also created a second fresh account, with verified Email (nothing else done), and get the exact same error.

Error is:

Running pokecli.py for count: 0
Traceback (most recent call last):
  File "pokecli.py", line 39, in <module>
    from pgoapi.exceptions import NotLoggedInException, ServerSideRequestThrottlingException, ServerBusyOrOfflineException
ImportError: No module named pgoapi.exceptions

This just repeats. Directly before and afterwards, the same (only checked with 1st) account is working on phone, in similar location.

### Other Information
OS: Windows 8.1 and Ubuntu 14.04, each same behavior, here are details for Win 8.1
Branch: dev or master, both checked, dev right now
Git Commit: d4200e9b34ef6ec9826eb81a6ceb420162b8c674
Python Version: Python 2.7.12 :: Anaconda 4.1.1 (64-bit)

**Comment by davidschoerkmayr** [comment]
same problem here.

**Comment by ThadeuBrito** [comment]
What happens if you comment or remove the line 38 from `pokecli.py`?
```python
from pgoapi.exceptions import NotLoggedInException, ServerSideRequestThrottlingException, ServerBusyOrOfflineException
```

**Comment by itsescde** [comment]
File "pokecli.py", line 39, in <module>
from geopy.exc import GeocoderQuotaExceeded
ImportError: No module named geopy.exc
Sat Aug 13 11:12:48 EDT 2016 Pokebot  Stopped.
Press any button or wait 20 seconds to continue.

**Comment by vi23** [comment]
try doing 
export PATH = "~/pokemon/PokemonGo-Bot/bin:$PATH"

**Comment by andirapandi** [comment]
@vi23 I cannot see a ./bin subfolder .
@ThadeuBrito Ok, I never properly read that I did not get the exception but that importing the exceptions failed. Once commenting out the line 39, I get an error on line 40. So I will have a look at where I can find those files/why they are missing.

**Comment by leanhdaovn** [comment]
Same issue.
Adding the bin folder to PATH (as @vi23 suggested) does not help.

**Comment by leanhdaovn** [comment]
Running this command fixed the problem for me
`source bin/activate`

**Comment by andirapandi** [comment]
It is working now.

My installation was incomplete.

I followed "Installation Windows" part on https://github.com/PokemonGoF/PokemonGo-Bot/blob/dev/docs/installation.md , but that was missing both protobuf and encrypt.so, as described above in the "Manual installation" section.

So for protobuf, I simply downloaded from the link given and extracted to c:\python27.

For encrypt.so, since only the source is linked with makefile and no easy way to build on Windows, I downloaded it as per comment in https://github.com/PokemonGoF/PokemonGo-Bot/issues/2923 from https://github.com/PokemonGoMap/PokemonGo-Map/tree/develop/pogom/libencrypt and then renamed it properly (encrypt_64.dll in my case) and put into the folder I laso checked out into.
